### ISSUE TITLE
Vzhled faset

### ISSUE THREAD

**Issue by annie-cz** [NLCR/SeznamDNNT#73]
Bylo by možné, aby se fasety, podle nichž lze filtrovat, překládaly všechny do nějakého lidštějšího a méně strojového jazyka?

Předně mrkám po fasetách "stav dokumentu", "formát dokumentu", "jazyk dokumentu" a "nakladatel" (tedy víceméně po všech). 

Ani já jako knihovník z hlavy neznám všechny sigly, natož průměrný uživatel/nakladatel. Totéž formát dokumentu - absolutně netuším o co se jedná. Ve stavu dokumentu to nevypadá hezky a nevím, proč by měl uživatel řešit jakkoli zkratky/kódová označení jednotlivých filtrů dokumentů. Totéž u jazyků - nevypadá to dobře a u některých jazyků lze mít pochybnosti (co je to třeba "mul"?). A obecně nevypadá dobře, když jsou nakladatelé a další organizace psané malým písmem.

**Comment by zabak** [comment]
Většinu těch textací lze přímo převzít z web klienta Krameria, kde je to redukované na skutečně používané údaje, nebo převzít kompletně příslušné databáze (zejm. sigly, jazyky). Ostatní filtry se dají otextovat krátkým výčtem.

**Comment by smallcubes** [comment]
Příbuzné issue #38

**Comment by JanMeritus** [comment]
@annie-cz @zabak aplikace je primarne pro specialisty aby mohli ridit politiky zverejneneni. Myslim ale ze konfigurovatelny prekladovy slovnik pro fazety:

sigla, jazyk, format dokumentu a stav dokumentu by mohl nekde byt. Data by se ale v zadnem pripade sami o sobe menit nemeli, jenom jejich zobrazeni pro uzivatele.

**Comment by annie-cz** [comment]
Ano, ale počítá se přeci i s tím, že tam poleze i běžný uživatel, nebo se pletu?

Jinak samozřejmě data měnit nechci, ale překlad by byl za mě velké dobro.

**Comment by pavel-stastny** [comment]
@JanMeritus @zabak @annie-cz  Počeštili jsme facety jazyk a sigla (Překlady se přebraly z web klienta), vyčistli jsme nakladatele (teď nemá čárku na konci) a přidali facet pro licence.  Ohledně stavů, v další verzi zkusíme sjednotit a polidštit stavy dokumentů aby bylo uživatelsky čitelné, například:  

1. Dílo na seznamu sdnnt
2. Dílo bude na seznamu sdnnt  - čekací lhůta končí dd.mm.yyyy
3. Dílo vyřazené 
4. Dílo volné 

PS: Nyní se reindexují data, cca hodinu bude obsah neúplný

**Comment by annie-cz** [comment]
A tohle je prosím potřeba, aby tam bylo? Co uživateli řekne, že je něco PA: ... nebo N: ... ? Nemůže tam být jenom text? Tohle vypadá strašně škaredě.

![image](https://user-images.githubusercontent.com/43028501/135825677-f5493fd2-f6e3-45b6-a836-c6da0bec2482.png)

**Comment by annie-cz** [comment]
Jinak proč se toto textově liší?

![image](https://user-images.githubusercontent.com/43028501/135825997-0fd4b9f8-7103-431b-884d-7d47f5a9e446.png)

**Comment by smallcubes** [comment]
aktuálně dochází ke změně vyhledávacího rozhraní viz #130

**Comment by JanMeritus** [comment]
@annie-cz prosim pak o reakci

**Comment by annie-cz** [comment]
Nemám jak to otestovat, neboť se nikam nedostanu.

_Soucasna verze -> https://sdnnt-dev.nkp.cz/sdnnt/search?page=0
Nova verze -> https://sdnnt-dev.nkp.cz/sdnnt-0.85_nl/search?page=0_

Mám přístup pouze na https://195.113.133.53/sdnnt/.

**Comment by zabak** [comment]
Nová verze je https://195.113.133.53/sdnnt-0.85_nl/search?page=0

**Comment by annie-cz** [comment]
OK, díky. :)

Vypadá to mnohem lip, nicméně co je prosím toto? To vypadá, jako něco zapomenutého.

![image](https://user-images.githubusercontent.com/43028501/141784588-b7fcba45-228c-4d59-95f8-f49cf75556e4.png)

**Comment by pkudela** [comment]
Dobry den @annie-cz, to co poslal pan Zabicka je vyvojova verze starsiho data, kterou jsem jeste vylepsil viz. issue #130 a #78. Tato verze bude nasazena na testovaci server jakmile bude hotove workflow na kterem dela kolega Pavel Stastny.

**Comment by annie-cz** [comment]
Dobrý den, díky za info. V tom případě ale nemohu adekvátně reagovat, protože se nikam nedostanu a přiznám se, že se začínám skutečně ztrácet v tom, co je a co není aktuální. Tj. prosím o zkvalitnění toku informací, neboť já, kolegyně Markéta Krutská i Petr Žabička máme evidentně úplně odlišné informace. Hodně špatně se pak reaguje i sleduje, v jakém stavu ta aplikace je.

Kromě toho se musím znovu zeptat - je nutné tam prosím mít ty zkratky "A", "NZ", atd.? Považuji sice za příjemné to označit barvami, tím se to rozhodně vylepšilo a je to přehlednější, ale co uživatel podobnými zkratkami získá? Kromě toho na první pohled ty zkratky ani nejsou logické, viz třeba PA = publikováno jako návrh. Šlo by prosím tohle všechno prostě vyloučit a nechat místo "PA - dílo publikováno jako návrh" nechat jen "publikováno jako návrh" nebo "navržené na seznam" nebo něco takového?

Omlouvám se, opakuju se s tím, ale prostě jsem trošku markeťák a zároveň pracuju jak s knihovníky, tak s uživateli a tohle jsou prostě informace a zkratky navíc, které dle mého názoru ničemu nepomůžou.

**Comment by pkudela** [comment]
Dobry den, ty zkratky nyni maji svuj nemaly vyznam. Nove jsou zkratky svazane s logikou noveho layoutu. Je to jakysi marker(ikona)/identifikator stavu, ktery zaroven identifikuje barevne dany facet s vysledky vyhledavani. S tim take souvisi nova home page viz. issue #78. V issue #130 si pak muzete precist vice o novem layoutu a duvodu proc muselo dojit k temto zmenam. Prosim, vyckejte na nasazneni nove verze na testovaci server kde budou zohledneny vsechny posledni zmeny tykajici se layoutu i jinych zmen, na kterych se pracuje. Dekuji za trpelivost a  pochopeni.
